bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/CredentialInterceptorCache.java

```diff
@@ -104,17 +104,13 @@ public synchronized ClientInterceptor getCredentialsInterceptor(
             credentials.getClass().getName()));
 
     RefreshingOAuth2CredentialsInterceptor oauth2Interceptor =
-        new RefreshingOAuth2CredentialsInterceptor(executor, (OAuth2Credentials) credentials,
-            retryOptions);
+        new RefreshingOAuth2CredentialsInterceptor(executor, (OAuth2Credentials) credentials);
 
     // The RefreshingOAuth2CredentialsInterceptor uses the credentials to get a security token that
     // will live for a short time.  That token is added on all calls by the gRPC interceptor to
     // allow users to access secure resources.
-    //
-    // Perform that token lookup asynchronously. This permits other work to be done in
-    // parallel. The RefreshingOAuth2CredentialsInterceptor has internal locking that assures that
-    // the oauth2 token is loaded before the interceptor proceeds with any calls.
-    oauth2Interceptor.asyncRefresh();
+    oauth2Interceptor.syncRefresh();
```

Comment:
it's not done on startup.  App engine always needs to do this synchronously, but that doesn't mean that we should reduce performance elsewhere for the 80%+ case, does it?